src/server/troubleshoot/index.md

```diff
@@ -17,3 +17,25 @@ Did you get an error that the server is not properly configured?
    
 2. Restart the container with the `MERGIN_BASE_URL` variable
 
+## Emails are not sent
+
+Mergin Maps server is sending emails to users in following cases:
+
+* Email verification
+* Password reset
+* Incoming access request
+* Invitation to workspace (only for <EnterprisePlatformName />)
+
```

Comment:
I'd prefer not mentioning the exact situations when we send emails. These can change and even the following list is not complete (it is missing the "account created" email).